FND/GoalsAndObjectives/Objectives.rdf

```diff
@@ -137,26 +156,21 @@ THE SOFTWARE IS PROVIDED &apos;AS IS&apos;, WITHOUT WARRANTY OF ANY KIND, EXPRES
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-gao-obj;Method">
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-gao-obj;hasObjective"/>
-				<owl:onClass rdf:resource="&fibo-fnd-gao-obj;Objective"/>
-				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
-			</owl:Restriction>
-		</rdfs:subClassOf>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Approach"/>
 		<rdfs:label>method</rdfs:label>
-		<skos:definition>systematic procedure or approach for doing or calculating something, consisting of defined steps or rules to achieve a result</skos:definition>
+		<skos:definition>approach distinguished by systematic, repeatable procedures and operational specificity for doing or calculating something, consisting of defined steps or rules to achieve a result</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-gao-obj;Objective">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Aim"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&cmns-dt;hasDatePeriod"/>
 				<owl:someValuesFrom rdf:resource="&cmns-dt;DatePeriod"/>
```

Comment:
normally an Objective has an end date not a period. Not sure it should be mandatory